js/private/js_binary.bzl

```diff
@@ -182,14 +182,16 @@ _ATTRS = {
         doc = """When True, data files and the entry_point file are copied to the Bazel output tree before being passed
         as inputs to runfiles.
 
-        Ideally, the default for this would be False as it is optimal, but there is a yet unresloved issue of ESM imports
+        Defaults to True so that a js_binary with the default value is compatible with js_run_binary with run_from_execroot
```

Comment:
backticks around the rule/method/param names?

Reply:
🍔